client/data/settings/reducer.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
```

Comment:
Sorry, nitpicking here... :innocent: can we rename this to something more verbose like `ACTION_TYPES`? We're not gonna use more constants in this file probably but I think being more verbose in naming is always good :smiley:

Reply:
Sure, done!